real_time/surface_src/java_src/Alice/build.gradle

```diff
@@ -50,7 +50,8 @@ dependencies {
     compile files('library/orsoncharts-1.4-eval-nofx.jar')
 
 
-    compile files('library/RXTXcomm.jar')
+    compile files('library/purejavacomm.jar')
```

Comment:
OK quick search shows the developer doesn't actually host on maven, there doesn't seem to be a way for us to do this through gradle, ok as is